package.json

```diff
@@ -1,20 +1,21 @@
 {
   "name": "@digicatapult/dscp-hyproof-api",
-  "version": "0.7.16",
+  "version": "0.7.17",
   "description": "An OpenAPI API service for DSCP",
   "main": "src/index.ts",
+  "type": "module",
   "scripts": {
     "lint": "eslint .",
     "depcheck": "depcheck",
     "build:tsoa": "tsoa spec-and-routes",
     "build": "tsoa spec-and-routes && tsc",
     "start": "node build/index.js",
-    "dev": "npm run build && NODE_ENV=dev concurrently \"npx tsc --watch\" \"nodemon -q build/index.js | pino-colada\"",
-    "test:offchain": "NODE_ENV=test ./node_modules/.bin/mocha --config ./test/mocharc.json -r ts-node/register ./test/integration/offchain/*.test.ts",
-    "test:onchain": "NODE_ENV=test ./node_modules/.bin/mocha --config ./test/mocharc.json -r ts-node/register ./test/integration/onchain/*.test.ts",
-    "test:unit": "NODE_ENV=test ./node_modules/.bin/mocha --config ./test/mocharc.json -r ts-node/register ./src/**/*.test.ts",
-    "db:migrate": "npx knex migrate:latest --knexfile src/lib/db/knexfile.ts",
-    "db:rollback": "npx knex migrate:rollback --knexfile src/lib/db/knexfile.ts",
+    "dev": "concurrently \"NODE_ENV=dev nodemon | pino-colada\" \"nodemon -x tsoa spec-and-routes --ignore src/routes.ts --ignore src/swagger.json\"",
+    "test:offchain": "NODE_ENV=test ./node_modules/.bin/mocha --config ./test/mocharc.json ./test/integration/offchain/*.test.ts",
```

Comment:
What about adding another ENV variable, would that work .. as in 

```
NODE_OPTIONS="--loader ts-node/esm --experimental-json-modules"
```

Reply:
These both appear as experimental options and don't seem to be where node is going unfortunately. https://github.com/nodejs/node/issues/51196